### ISSUE TITLE
Nested list example not working

### ISSUE THREAD

**Issue by Sabrinovsky** [computerjazz/react-native-draggable-flatlist#530]
**Describe the bug**
The nested example from the readme is not working: https://snack.expo.dev/@computerjazz/draggable-flatlist-examples

**To Reproduce**
Access the link example. Not working on android physical and emulator.

**Comment by codeandcoffee-io** [comment]
Upvote - tried the snack today and it simply crashes in expo go

**Comment by alexfoxy** [comment]
Yep, not working! Upgraded all the packages and crashes on device.

**Comment by arpitv96171** [comment]
it is most likely that the snack example is using expo > 48 on the website and the example is using expo 48, which can cause this.
you will have to set expo version to 48 to be able to run it.

as for mobile, expo snack needs to be a certain version (max supported expo go app version 2.28.7) to be able to run expo 48 code, which I'm not sure is possible.

**Comment by arpitv96171** [comment]
you can very well test it out on simulators by following below steps:

1. Delete `yarn.lock` and `node_modules`
2. Run `yarn` within `Example` folder
3. Run `npx expo install --fix` within `Example` folder
4. Wrap root app in `<GestureHandlerRootView style={{ flex: 1 }}></GestureHandlerRootView>` in the file `Example/App.tsx`
5. Change `makeUrl` to `createUrl` in `Example/navigation/LinkingConfiguration.ts`
6. Run `yarn ios`
